elements/hpc-hypervisor/pkgs

```diff
@@ -0,0 +1 @@
+qemu-kvm-ev
```

Comment:
Do we need this and the one above? I think the easiest way to install packages is to add a package-installs.yaml to the element root, like this:

`qemu-kvm-ev:`

See https://docs.openstack.org/developer/diskimage-builder/elements/package-installs/README.html.